### PR TITLE
multi-region-api updates

### PR DESCRIPTION
**Pull request by markjbrown**
# PR Checklist

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

Updates to:
- api-version
- naming in bicep file
- system managed failover
- bounded staleness max values
- doc owner

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by markjbrown** [comment]
@bmoore-msft Brian, this job keeps failing because I have an empty property in my template. This template has never failed before so it appears the test has changed.

Is there an easy way to handle this? This template was designed to allow for people to select the database API that results in a combination of kind and capabilities properties being hydrated. SQL API does not have any capabilities which is why it is empty.

If there's a way to mark this template so it can skip this test that would be helpful. If not, then may need to delete the sample.

Thanks.

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.documentdb/cosmosdb-create-multi-region-account/main.bicep [hunk 1]
```diff
@@ -30,30 +30,30 @@ param api string = 'Sql'
 ])
 param defaultConsistencyLevel string = 'Session'
 
-@description('Max stale requests. Required for BoundedStaleness. Valid ranges, Single Region: 10 to 1000000. Multi Region: 100000 to 1000000.')
+@description('Max stale requests. Required for BoundedStaleness. Valid ranges, Single Region: 10 to 2,147,483,647. Multi Region: 100,000 to 2,147,483,647.')
```
Comment (bmoore-msft) [inline quickstarts/microsoft.documentdb/cosmosdb-create-multi-region-account/main.bicep:33]: I suspect you used the commas for readability ```100,000``` but there will be users who put in the commas for the values and that will fail...

Comment (markjbrown) [reply quickstarts/microsoft.documentdb/cosmosdb-create-multi-region-account/main.bicep:33]: Yes, I did. All of the updates I added commas. You've already merged some of these. Do you want me to submit another PR to remove?

I also need your help in figuring out a path for this empty parameter. I've tried a bunch of different things including passing an empty string into Capabilities or just removing it altogether. Not sure how to get this to work.

Comment (bmoore-msft) [reply quickstarts/microsoft.documentdb/cosmosdb-create-multi-region-account/main.bicep:33]: re: the commas, I don't think that's urgent, I suspect the UX in portal will flag them, so it's mostly frustrating in that case...

re: the empty param maybe ping offline for that one.

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@markjbrown - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**